scripts/examples/gn_p6_example.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+#
+#    Copyright (c) 2021 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+set -e
+# Build script for GN P6 examples GitHub workflow.
+source "$(dirname "$0")/../../scripts/activate.sh"
+
+# Install required software
+if [ -d "/opt/ModusToolbox" ]; then
+    export CY_TOOLS_PATHS="/opt/ModusToolbox/tools_2.3"
```

Comment:
Why does this need it to be in /opt ? None of the other platforms are that stringent about where the tools are installed.

Do you intend this to ever work on Windows or is that not supported in the first place?